android/app/src/main/jni/webrtc.c

```diff
@@ -498,6 +497,7 @@ on_server_message (SoupWebsocketConnection * conn, SoupWebsocketDataType type,
       case PEER_CONNECTED:
       case PEER_CALL_NEGOTIATING:
         webrtc->app_state = PEER_CALL_ERROR;
+            break;
```

Comment:
bad indentation, ideally we would install a pre-commit hook in there, similar to the one installed in gstreamer: https://gitlab.freedesktop.org/gstreamer/gstreamer/-/blob/master/hooks/pre-commit.hook

Reply:
is it original indentation

Reply:
hm true, that's how the cases above are indented, kind of a mess really, whatever :)